int/api/utils.go

```diff
@@ -1,29 +1,26 @@
 package api
 
 import (
-	"html/template"
+	"mime"
 	"net/http"
 	"path/filepath"
+	"text/template"
 
 	"github.com/go-openapi/runtime"
 )
 
 func contentType(rsc string) map[string]string {
 	var contentType map[string]string
 
-	switch filepath.Ext(rsc) {
-	case ".css":
-		contentType = map[string]string{"Content-Type": "text/css"}
-	case ".js":
-		contentType = map[string]string{"Content-Type": "text/javascript"}
-	case ".html":
-		contentType = map[string]string{"Content-Type": "text/html"}
-	case ".webp":
-		contentType = map[string]string{"Content-Type": "text/webp"}
-	case ".png":
-		contentType = map[string]string{"Content-Type": "image/png"}
-	default:
-		contentType = map[string]string{}
+	if filepath.Ext(rsc) == ".otf" {
```

Comment:
You should create a variable to store `filepath.Ext(rsc)`.
It will be clearer if we have other edge cases